2022-03-28 15:16:59,581 INFO    MainThread:17657 [wandb_setup.py:_flush():71] setting env: {}
2022-03-28 15:16:59,581 INFO    MainThread:17657 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-28 15:16:59,581 INFO    MainThread:17657 [wandb_init.py:_log_setup():371] Logging user logs to /Users/kiranganeshan/reading-group/wandb/run-20220328_151659-1ial2dv1/logs/debug.log
2022-03-28 15:16:59,582 INFO    MainThread:17657 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/kiranganeshan/reading-group/wandb/run-20220328_151659-1ial2dv1/logs/debug-internal.log
2022-03-28 15:16:59,582 INFO    MainThread:17657 [wandb_init.py:init():404] calling init triggers
2022-03-28 15:16:59,582 INFO    MainThread:17657 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-28 15:16:59,582 INFO    MainThread:17657 [wandb_init.py:init():449] starting backend
2022-03-28 15:16:59,582 INFO    MainThread:17657 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-28 15:16:59,596 INFO    MainThread:17657 [backend.py:ensure_launched():199] starting backend process...
2022-03-28 15:16:59,603 INFO    MainThread:17657 [backend.py:ensure_launched():205] started backend process with pid: 17671
2022-03-28 15:16:59,605 INFO    MainThread:17657 [wandb_init.py:init():458] backend started and connected
2022-03-28 15:16:59,609 INFO    MainThread:17657 [wandb_init.py:init():519] updated telemetry
2022-03-28 15:16:59,623 INFO    MainThread:17657 [wandb_init.py:init():549] communicating current version
2022-03-28 15:17:00,210 INFO    MainThread:17657 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-28 15:17:00,210 INFO    MainThread:17657 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-03-28 15:17:00,421 INFO    MainThread:17657 [wandb_init.py:init():592] starting run threads in backend
2022-03-28 15:17:02,528 INFO    MainThread:17657 [wandb_run.py:_console_start():1816] atexit reg
2022-03-28 15:17:02,529 INFO    MainThread:17657 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-03-28 15:17:02,530 INFO    MainThread:17657 [wandb_run.py:_redirect():1695] Redirecting console.
2022-03-28 15:17:02,533 INFO    MainThread:17657 [wandb_run.py:_redirect():1751] Redirects installed.
2022-03-28 15:17:02,534 INFO    MainThread:17657 [wandb_init.py:init():619] run started, returning control to user process
2022-03-28 15:17:02,681 INFO    MainThread:17657 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-03-28 15:17:02,681 INFO    MainThread:17657 [wandb_run.py:_restore():1758] restore
2022-03-28 15:17:05,067 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 690
}

2022-03-28 15:17:05,173 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 690
}

2022-03-28 15:17:05,418 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 690
}

2022-03-28 15:17:05,522 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4713
}

2022-03-28 15:17:05,624 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 690
  total_bytes: 4713
}

2022-03-28 15:17:05,730 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}

2022-03-28 15:17:05,834 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}

2022-03-28 15:17:05,940 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}

2022-03-28 15:17:06,045 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}

2022-03-28 15:17:06,151 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}

2022-03-28 15:17:06,612 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}

2022-03-28 15:17:06,950 INFO    MainThread:17657 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4713
  total_bytes: 4713
}
local_info {
}

2022-03-28 15:17:08,082 INFO    MainThread:17657 [wandb_run.py:_append_files():2184] logging synced files
